net/freeradius/src/opnsense/mvc/app/models/OPNsense/Freeradius/User.xml

```diff
@@ -15,7 +15,7 @@
                 </username>
                 <password type="TextField">
                     <Required>Y</Required>
-                    <mask>/^([0-9a-zA-Z._\-\!\$\%\/\(\)\+\#\=\{\}]){1,128}$/u</mask>
+                    <mask>/^([0-9a-zA-Z._\-\,\£\$\%\^\&amp;\*\(\)\}\{\@\:\'\-\=\-\_\+\-\¬\`\#\~\?\&#60;\>\,\;\@\!\$\%\/\(\)\+\#\=\{\}]){1,128}$/u</mask>
```

Comment:
if we exclude double-quote only wouldn't that suffice? making it bigger doesn't make it more maintainable andthere are still character holes